Source/Urho3D/Urho2D/SpriteSheet2D.cpp

```diff
@@ -51,6 +52,59 @@ void SpriteSheet2D::RegisterObject(Context* context)
     context->AddFactoryReflection<SpriteSheet2D>();
 }
 
+bool SpriteSheet2D::Save(Serializer& dest) const
+{
+    // Only XML format supported at the moment.
+    SharedPtr<XMLFile> xml(MakeShared<XMLFile>(context_));
+    XMLElement rootElem = xml->CreateRoot("TextureAtlas");
+    Save(rootElem);
+    return xml->Save(dest);
+}
+
+bool SpriteSheet2D::Save(XMLElement& rootElem) const
+{
+    if (rootElem.IsNull())
+    {
+        URHO3D_LOGERROR("Can not save SpriteSheet2D to null XML element");
+        return false;
+    }
+
+    if (texture_)
+    {
+        ea::string imagePath;
+        if (Urho3D::GetRelativePath(Urho3D::GetParentPath(GetName()), texture_->GetName(), imagePath))
```

Comment:
Why is this needed and what does it do? Looks odd. Seems like `texture_->GetName()` should always be enough.

Reply:
It seems it is existing file format with a relative path to the texture: https://doc.starling-framework.org/v2.0/starling/textures/TextureAtlas.html

Reply:
Oh, this is implementing a third-party format. OK then, in that case my previous comment is not valid regarding new serialization. 👍